streaming/flink/helm-values/prom-grafana-values.yaml

```diff
@@ -0,0 +1,21 @@
+prometheus:
+  prometheusSpec:
+    serviceMonitorSelectorNilUsesHelmValues: false
+    podMonitorSelectorNilUsesHelmValues: false
+    serviceMonitorSelector: {}
+    serviceMonitorNamespaceSelector: {}
+grafana:
```

Comment:
Did you mean to update this config in the above file ? `streaming/flink/helm-values/kube-prometheus.yaml` This file will be used as a default while deploying Kube Prometheus stack.

Reply:
Not required already there